prbt_hardware_support/include/prbt_hardware_support/libmodbus_client.h

```diff
@@ -60,6 +60,7 @@ class LibModbusClient : public ModbusClient
 
 private:
   modbus_t* modbus_connection_{ nullptr };
+  bool checkIPConnection(const char* ip, unsigned int port, unsigned int timeout, unsigned int retries);
```

Comment:
Does not need to be member. But if it could be static and const.

Reply:
What do you mean exactly?

Reply:
As far as I understand this function does not use any member variables and thus does not need to be a member function.

Reply:
Thank you. Ok, I made it constant. Its pushed already.